loader.js

```diff
@@ -949,6 +1004,63 @@ var Module = null;
      return { extra_np2_args: args };
    };
 
+   /**
+    * V86Loader
+    */
+    function V86Loader() {
+      var config = Array.prototype.reduce.call(arguments, extend);
+      config.memory_size = 32 << 20;
```

Comment:
This overwrites the memory_size with a fixed value; probably not what you want. Why not make the memory_size function look like this:

```
return {"memory_size": amount || 32 << 20};
```

Reply:
You might also consider changing it to accept values in megabytes rather than bytes.

Reply:
What guarantees the memorySize function will be called in non-IA mode (see `example_v86.html`)? Do we require the user to call it in such a scenario? I wanted to avoid that.

Reply:
Here's a different proposal still, trying to keep the default in the constructor for the above-mentioned reason. I've changed it to accept values in megabytes, however with support for fractional values (like `0.5`; I doubt its usefulness, but `vga_memory_size` might benefit).

Reply:
That works too. Why do you doubt its usefulness? Oh, you just doubt that using fractional megabytes is useful. You are probably right; nobody is going to do 23.5 MB, although apparently the emulator can deal with it since it takes bytes.

Reply:
Yeah, but someone might want 512 KB of VGA memory. That was my thinking.